DTS-STN/Alpha-Site
# Bug/344 error on click

# Description

[Signup error click](https://trello.com/c/Uo1X4ZYo/344-344-fix-onclick-is-not-a-function-when-user-clicks-on-error-message)

When clicking on an item from the error summary for teh signup form, the app would crash - see Trello board item linked above.

## In This PR

Added the functionality to scroll the user to the desired error.
Added smooth scrolling behaviour to both, the sign up button, and the error summary items.

## Acceptance Criteria

N/A

## Test Instructions

1. `git checkout Bug/344-ErrorOnClick` and `npm run dev`
2.  Navigate to the sign up page.
3.  Scroll all the way down and press on the "Submit" button.
4. See the scrolling animation
5. Press on all 4 of the error items to see where the page is scrolled to.

## Help Requested

- Any constructive feedback is more than welcome!

## Checklist
